### DISCUSSION TITLE
Integrate Centreon with Stackstorm

### DISCUSSION THREAD

**Discussion by Chaimaa-Br** [StackStorm/st2#6019] [Packs] [answered]
Hello,
Is it possible to monitor and supervise Infrastruture on **Centreon** using Stackstom? (Meaning have a trigger that listens on the newly created alerts, actions to execute on Centreon, etc)

**Comment by nzlosh** [comment]
If it's got an API or Python SDK it should be possible to code an integration.

**Reply by Chaimaa-Br** [reply, marked as answer]
Yes, it does have an API that we could access through Rest API. Someone also said that since Centreon is compatible with Nagios, we could make some use of the Nagios pack mechanisms to be able to access Centreon's API. I'll try both ways and see which one is better.
